### DISCUSSION TITLE
¿Porqué después de suscribirme a un plan, me dice que este esta inactivo?

### DISCUSSION THREAD

**Discussion by mediosapoyoconservacion** [opengisch/QField#3978] [Bugs]
Hola, después de suscribir el plan a pro, este me aparece como inactivo, cuando la fecha entrar en servicio es el 22 de febrero de 2023.
Esto a que es debido.
![Plan pro suscrito Qfield Cloud](https://user-images.githubusercontent.com/126456315/221522686-4ce3bb60-8fee-413e-acc9-f429ac526036.png)

**Comment by cez2020** [comment]
Hola, has podido resolver este problema?

**Reply by mediosapoyoconservacion** [reply]
No. Hemos cambiado de plan, y hemos introducido todos los datos. Pero seguimos igual.
